apps/health_alerts/tests/test_views.py

```diff
@@ -0,0 +1,68 @@
+from http import HTTPStatus
+
+from django.contrib.messages import get_messages
+from django.urls import reverse
+
+import pytest
+from pytest_django.asserts import assertTemplateUsed
+
+
+@pytest.fixture
+def sign_up_data():
+    return dict(
+        personal_first_name="SpongeBob",
+        personal_last_name="SquarePants",
+        personal_medical_expertise="Sutures",
+        personal_professional_license="1294dk",
+        agency_name="The Crabby Patty",
+        agency_type="PDPH",
+        agency_zip_code="12340",
+        agency_position="Manager",
+        network_email="crabbypatty@gmail.com",
+        network_fax="(201) 555-0123",
+    )
+
+
+@pytest.fixture
+def url():
+    return reverse("health_alerts_sign_up")
+
+
+def test_get_sign_up_page(db, settings, client, url):
+    settings.STATICFILES_STORAGE = (
```

Comment:
You should be able to add this change to your dev settings and then remove the settings fixture here:
https://github.com/caktus/philly-hip/pull/81/files?file-filters%5B%5D=.py#diff-f35ca6429ddf447226fc2ff1342c932f17948d02069c2f59819f9af4c28273fa

Reply:
done